### PR TITLE
Feature/rw 7034 update

### PR DESCRIPTION
**Pull request by aniamy**

### CODE REVIEW

File: src/bookmarklet/inject_base.js [hunk 1]
```diff
@@ -193,11 +193,15 @@ translate.setHandler("attachmentProgress", function(obj, attachment, progress) {
 translate.setHandler("done", function(obj, returnValue) {
 	if(returnValue && haveItem) {
 		Zotero.ProgressWindow.startCloseTimer(2500);
+		cleanup();
 	} else if(!cancelled) {
-		new Zotero.ProgressWindow.ErrorMessage("translationError");
-		Zotero.ProgressWindow.startCloseTimer(8000);
+		Zotero.ProgressWindow.changeHeadline("We could not extract the metadata.");
```
Comment (cano-endava) [inline src/bookmarklet/inject_base.js:198]: looks like this message won't be translated, while the rest of STR does support different languages.

Comment (aniamy) [reply src/bookmarklet/inject_base.js:198]: Can you point me to a PME text that gets translated?
This is a Zotre progress window headline.

Comment (aniamy) [reply src/bookmarklet/inject_base.js:198]: Example: The current error that happened was a PME error. Should not have been translated from what I know.

Comment (cano-endava) [reply src/bookmarklet/inject_base.js:198]: You are right, PME != STR. STR is the one that has translations.

**Review by cano-endava — APPROVED** [review]

**Review by OhadKim — APPROVED** [review]

**Pull request merged**